client/extensions/woocommerce/app/settings/navigation.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { getSiteSlug } from 'state/sites/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import NavItem from 'components/section-nav/item';
+import NavTabs from 'components/section-nav/tabs';
+import SectionNav from 'components/section-nav';
+
+export const StoreSettings = ( {
+	siteId,
+	siteSlug,
+	translate,
+} ) => {
+	const pathSuffix = siteSlug ? '/' + siteSlug : '';
+	const filters = [];
+
+	filters.push( {
+		id: 'payments',
+		route: '/store/settings/payments' + pathSuffix,
+		title: translate( 'Payments' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **43** times:
`translate( 'Payment' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).